### ISSUE TITLE
Support all remaining segment displays.

### ISSUE THREAD

**Issue by freezy** [freezy/VisualPinball.Engine#298]
There's a plethora of segment/period/comma combinations we currently don't support.

- There's versions that only have a period (dot).
- Some have only a comma
- Some have a comma with two segments that can be both commas and periods.

Here's a pic of such a display:

[![image](https://user-images.githubusercontent.com/70426/116748250-71724180-a9ff-11eb-85b8-a8c669da9848.png)](http://www.bigdaddy-enterprises.com/ProductPages/XPINWilliamsDisplayBoards.html)

These are all the types that need support:

https://github.com/vpinball/pinmame-dotnet/blob/115d2e262a949c9e68e0c9b822b042dfea6cc398/src/PinMame/PinMameApi.cs#L54-L74

**Comment by freezy** [comment]
```glsl

float draw_circle(vec2 coord, float radius) {
    return step(length(coord), radius);
}

vec2 translate(vec2 coord, vec2 translate) {
    return coord - translate;
}

void mainImage( out vec4 fragColor, in vec2 fragCoord )
{
    vec2 uv = fragCoord/min(iResolution.x, iResolution.y);
   
    float circle = draw_circle(translate(uv, vec2(1.0, 0.5)), 0.3);    
    circle *= (1. - draw_circle(translate(uv, vec2(0.5, 0.5)), 0.6));
    circle *= (1. - draw_circle(translate(uv, vec2(1.2, 0.7)), 0.25));

    
    vec3 color = vec3(circle);
    fragColor = vec4(color, 1.0); 
}
```

![image](https://user-images.githubusercontent.com/70426/116822645-2df31100-ab80-11eb-8238-5dd91c4c6cf3.png)

**Comment by freezy** [comment]
Fixed in #300,
